src/components/source_editor/enums.js

```diff
@@ -0,0 +1,8 @@
+import {Made} from "@exabyte-io/made.js";
+import {displayMessage} from "../../i18n/messages";
```

Comment:
Should be absolute import - from top-level dir in `/imports/i18`

Reply:
Does not work in materials-designer `Module not found: You attempted to import /i18n/messages which falls outside of the project src/ directory. Relative imports outside of src/ are not supported`

Reply:
Isn't this only valid syntax for a meteor application?

Reply:
<img src="https://media3.giphy.com/media/99K3MhaQUNgfTR6tiS/giphy.gif"/>

Reply:
Sorry